docs/cloud/README.md

```diff
@@ -0,0 +1,32 @@
+# FlowForge Cloud
+
+FlowForge Cloud is a hosted service allowing users to sign-up and start creating teams and projects without having to install and manage their own instance of FlowForge.
+The [Concepts](../user/concepts.md) remain the same, however FlowForge Inc. is the administrator of the platform.
+
+## Billing
+Customers are billed at the team level for each project they create. This is a recurring monthly charge.
+See the [Billing](./billing.md) page for more detailed answers about billing.
+
+## Support
+Customers can get support by emailing [support@flowforge.com], we presently only offer support for the flowforge application and your account, any issues relating to Node-RED such as your flows or a 3rd party node should be raised in the appropriate community forum, for example [https://discourse.nodered.org/] or the GitHub project of the third party node.
```

Comment:
Missing link location for our email address, did you intent to create a mailto://?

Reply:
Same for the discourse link.

Reply:
GitHub takes care of formatting emails/urls without any special markdown so have removed the. []